decoder.go

```diff
@@ -144,6 +144,11 @@ func (d *Decoder) Decode(v interface{}) (err error) {
 		n, err = d.ReadInt16()
 		rv.SetInt(int64(n))
 		return
+	case *int32:
```

Comment:
Going to merge, it's all good. Just curious, where it failing to decode a `int32`?

Reply:
in parsing table content.